packages/core/admin/admin/src/utils/uniqueAdminHash.js

```diff
@@ -17,8 +17,6 @@ export default async function hashAdminUserEmail(payload) {
   try {
     return await utils.digestMessage(payload.email);
   } catch (error) {
-    const hash = await import('hash.js');
```

Comment:
you removed the code but I don't see the package being removed from the dependencies ?